recipes/giddy/meta.yaml

```
@@ -0,0 +1,53 @@
+
+{% set name = "giddy" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c284856579ab95ff6a63b6a7c838ab0a48c8669745ea16f8428a012abedb750
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >= 3
```

giddy has not been tested against python 3.7 
[only 3.5 and 3.6](https://github.com/pysal/giddy/blob/master/.travis.yml#L6) at the moment
but I assume it is OK not to specify it here?

---

Couldn't you just test it? As it is a `noarch: python` package it should work and we're safe here with `python >= 3`.

---

sounds like we're ok on this?

---

The space between the operator and the number needs to be removed i.e `python >=3` (The space before `>=` is important)
```suggestion
    - python >=3
```
```suggestion
    - python >= 3
```